ADJ-Internship/BusinessService/Implementations/ProgressCheckService.cs

```diff
@@ -0,0 +1,204 @@
+﻿using ADJ.BusinessService.Core;
+using ADJ.BusinessService.Dtos;
+using ADJ.BusinessService.Interfaces;
+using ADJ.Common;
+using ADJ.DataModel.OrderTrack;
+using ADJ.Repository.Core;
+using ADJ.Repository.Interfaces;
+using AutoMapper;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ADJ.BusinessService.Implementations
+{
+    public class ProgressCheckService : ServiceBase, IProgressCheckService
+    {
+        private readonly IDataProvider<Order> _odDataProvider;
+        private readonly IOrderRepository _odRepository;
```

Comment:
make the variable name clearer, _orderDataProvider
or if it is too long, try _progressCheckDataPrd, _progressCheckRepo, ...